### DISCUSSION TITLE
hidden files

### DISCUSSION THREAD

**Discussion by houshao666** [DirectoryLister/DirectoryLister#570] [Help & Support] [answered]
Hello! I want to hide. ' nginx.htaccess 'and' README.md I don't understand PHP, can you tell me how to set it
你好！我想隐藏'nginx.htaccess'和'README.md',我不懂php，能告诉我怎么设置吗

**Comment by PHLAK** [comment, marked as answer]
To hide files, start by creating a file named `.hidden` in the root of your listing. Then, in this file you can add the following entries

```
nginx.htaccess
README.md
```

See the [Hiding Files](https://docs.directorylister.com/configuration/hiding-files) documentation for additional information about hiding files and file ignore patterns.

**Reply by Omen128** [reply]
I have a lot of files and folders in my folder, how can I display only the files and folders I need? I tried to add [^111] in the file. hidden - it worked, but only for one file...

**Reply by PHLAK** [reply]
Unfortunately hiding files works like a blocklist (only _hide_ the files specified) and there is no way to reverse that to act like an allowlist (only _show_ the files specified).

**Reply by Omen128** [reply]
How can I create allowlist so that I can show, for example, only three files (test01.html, test02.html, test03.html) ?
Found in the documentation only Hiding Files...

**Reply by PHLAK** [reply]
This is not currently possible.
